Extras/FrontendTests/package.json

```diff
@@ -4,13 +4,14 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "npx playwright test"
+    "test": "npx playwright test",
+    "build": ""
```

Comment:
You may notice a few empty build scripts in this commit.

Those were added to enable running `npm run build` in the root of the repo and building all workspaces - handy.